sdv/single_table/utils.py

```diff
@@ -349,3 +349,23 @@ def log_numerical_distributions_error(numerical_distributions, processed_data_co
             f"cannot be applied to column '{column}' because it no longer "
             'exists after preprocessing.'
         )
+
+
+class GANMixin:
```

Comment:
How do you feel about putting this class in the ctgan file since all the gans are already there? Also I think VAEs aren't technically GANs. Maybe we should change the name?